CactbotOverlay/CactbotEventSource.cs

```diff
@@ -157,58 +194,54 @@ public CactbotOverlay(CactbotOverlayConfig config)
       fast_update_timer_.Interval = kFastTimerMilli;
       fast_update_timer_.Start();
 
-      if (is_first_overlay_initialized_) {
-        is_first_overlay_initialized_ = false;
-
-        var versions = new VersionChecker(this);
-        Version local = versions.GetLocalVersion();
-        Version remote = versions.GetRemoteVersion();
-
-        Version overlay = versions.GetOverlayPluginVersion();
-        Version ffxiv = versions.GetFFXIVPluginVersion();
-        Version act = versions.GetACTVersion();
+      var versions = new VersionChecker(this);
+      Version local = versions.GetLocalVersion();
+      Version remote = versions.GetRemoteVersion();
+
+      Version overlay = versions.GetOverlayPluginVersion();
+      Version ffxiv = versions.GetFFXIVPluginVersion();
+      Version act = versions.GetACTVersion();
+
+      // Print out version strings and locations to help users debug.
+      LogInfo("cactbot: {0} {1}", local.ToString(), versions.GetCactbotLocation());
+      LogInfo("OverlayPlugin: {0} {1}", overlay.ToString(), versions.GetOverlayPluginLocation());
+      LogInfo("FFXIV Plugin: {0} {1}", ffxiv.ToString(), versions.GetFFXIVPluginLocation());
+      LogInfo("ACT: {0} {1}", act.ToString(), versions.GetACTLocation());
+      if (language_ == null) {
+        LogInfo("Language: {0}", "(unknown)");
+      } else {
+        LogInfo("Language: {0}", language_);
+      }
 
-        // Print out version strings and locations to help users debug.
-        LogInfo("cactbot: {0} {1}", local.ToString(), versions.GetCactbotLocation());
-        LogInfo("OverlayPlugin: {0} {1}", overlay.ToString(), versions.GetOverlayPluginLocation());
-        LogInfo("FFXIV Plugin: {0} {1}", ffxiv.ToString(), versions.GetFFXIVPluginLocation());
-        LogInfo("ACT: {0} {1}", act.ToString(), versions.GetACTLocation());
-        if (language_ == null) {
-          LogInfo("Language: {0}", "(unknown)");
+      if (remote.Major == 0 && remote.Minor == 0) {
+        var result = System.Windows.Forms.MessageBox.Show(
+          "Github error while checking Cactbot version. " +
+          "Your current version is " + local + ".\n\n" +
+          "Manually check for newer version now?",
+          "Cactbot Manual Check",
+          System.Windows.Forms.MessageBoxButtons.YesNo);
+        if (result == System.Windows.Forms.DialogResult.Yes)
+          System.Diagnostics.Process.Start(VersionChecker.kReleaseUrl);
+      } else if (local < remote && false) {
```

Comment:
I need to remove the `&& false` here. Maybe we could disable the update check for debug builds?
Shouldn't be an issue for me anymore though since I've rebased against the current version.